test-dts/wrapper.d-test.ts

```diff
@@ -14,6 +14,9 @@ const domWrapper = wrapper.find('#other')
 let inputMaybe = wrapper.find('input')
 expectType<HTMLInputElement | undefined>(inputMaybe.element)
 
+let buttonComp = mount(defineComponent({ template: '<button>Click me</button>' }))
+expectType<HTMLButtonElement>(buttonComp.element)
+
```

Comment:
Do we have a test for the case with multiple root elements?

Reply:
Added a type test and here's an actual test for it: https://github.com/vuejs/test-utils/blob/main/tests/element.spec.ts#L46